.github/workflows/python-package.yml

```diff
@@ -19,7 +19,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pandas numpy matplotlib seaborn scikit-learn  
-    - name: test script
+        pip install pandas numpy matplotlib seaborn scikit-learn coverage  
+    - name: coverage script
```

Comment:
And this works the same way as running tests?
I.E git actions can use `coverage` to check if tests pass?

Reply:
What I'm getting at is a priori I would have expected to need a step for tests and another step for coverage

But if it all works with coverage thats great

Reply:
yup, it just runs the test file via the coverage command so it runs tests + checks for coverage

Reply:
okay perfect